app/assets/stylesheets/helpers/_header.scss

```diff
@@ -104,15 +105,18 @@
   .search-toggle {
```

Comment:
What do you think about making the styles for `.search-toggle` mobile first so the `display: none` would kick in `($from: tablet)` instead of being overwritten?

Reply:
Hi @injms that's a good point – @chris-gds has actually mentioned this too 👍. 

My rationale here is that the aim of this piece of work is to fix the issue by replacing the old media query style with the `govuk-media-query` equivalent.

This [other PR]( https://github.com/alphagov/static/pull/2293) removes a whole bunch of CSS and refactors the code so that it is mobile first, so I simply didn't want to repeat that on here in the interest of minimising potential conflicts/repetition (but I suppose there will be some conflicts either way, so I don't know... let me know what you think)